components/ui/SocialMediaItem.jsx

```diff
@@ -0,0 +1,11 @@
+const SocialMediaItem=({url,title,icon})=>{
+  return(
+    <li><a href={url} title={title} >
```

Comment:
Some indentation would make this file more readable:
```
 return (
    <li>
      <a>
        <img />
      </a>
    </li>
  );
```

Reply:
Done